Makefile

```
@@ -43,12 +43,15 @@ help: ## Shows this help message
 	@echo -e This is the farmvibes.ai makefile. Supported targets are:\\n
 	@grep -E -h '\s##\s' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}'
 
-local: cluster local-rest-api local-cache local-worker local-orchestrator local-data-ops ## Builds all images locally and deploys them into the local farmvibes.ai cluster
+local: cluster restore-git-lfs local-rest-api local-cache local-worker local-orchestrator local-data-ops ## Builds all images locally and deploys them into the local farmvibes.ai cluster
 	[ -z $(WAIT_AT_THE_END) ] || kubectl delete pods -l backend=terravibes && \
 		kubectl wait --for=condition=Available deployment --timeout=300s -l backend=terravibes
 
 revert: cluster revert-rest-api revert-cache revert-worker revert-orchestrator ## Reverts all images to the official version
 
+restore-git-lfs:
+	git lfs pull
```

How about something along the lines of the following?

```suggestion
git lfs pull || echo "git lfs was not found. Please see https://git-lfs.com/ to install it." && exit 1
```